Apps/phsymantecmc/symantecmanagementcenter_connector.py

```diff
@@ -470,12 +570,28 @@ def _handle_remove_url(self, param):
 
         policy_details = policy_details['policies'][0]
 
-        if policy_details['contentType'] != 'LOCAL_CATEGORY_DB':
-            message = 'Unable to edit policy, wrong content type. Received: {0}, expecting LOCAL_CATEGORY_DB'.format(policy_details['contentType'])
+        if policy_details['contentType'] not in ['LOCAL_CATEGORY_DB', 'URL_LIST', 'IP_LIST']:
```

Comment:
Since the list is used multiple times, it may be worthwhile to make it a constant in the consts.py file

Reply:
Fixed in #369